internal/resources/resource_instances_helper.go

```diff
@@ -147,6 +147,7 @@ func getInstanceDefaultSchema(isClone bool) *schema.Resource {
 							Optional: true,
 							Description: `Storage type ID can be obtained from hpegl_vmaas_instance_disk_type
 							data source.`,
+							DiffSuppressFunc: utils.SkipField(),
```

Comment:
needed cause 
![image](https://github.com/user-attachments/assets/2c6edb43-3568-4d8b-856e-a0e71ca38d4b)

Reply:
The skip diff is needed cause, when a vol is created without providing `storage_type` then null will be present in main.tf, but upon creating the statefile gets id 1 which is default type. And then when an additional vol is added, then comparing from main.tf and statefile it detects a change in `storage_type` and throws err